stdlib/urllib/parse.pyi

```diff
@@ -169,20 +193,32 @@ def urljoin(base: AnyStr, url: AnyStr | None, allow_fragments: bool = True) -> A
 def urlparse(url: str, scheme: str | None = "", allow_fragments: bool = True) -> ParseResult: ...
 @overload
 def urlparse(
-    url: bytes | bytearray | None, scheme: bytes | bytearray | None = ..., allow_fragments: bool = True
+    url: bytes | bytearray | None, scheme: bytes | bytearray | None, allow_fragments: bool = True
 ) -> ParseResultBytes: ...
 @overload
+def urlparse(
```

Comment:
What does this overload cover? We should reject mixed calls, since those throw an error. We should also remove `| None` from the first overload; passing None always fails.

Reply:
The original version of this PR didn't have this overload, but I was pretty scared by the size of the mypy_primer diff: https://github.com/python/typeshed/pull/9629#issuecomment-1409519042.

So I figured I'd try adding a fallback overload of some kind. But I agree it doesn't make much sense.

Reply:
Hm, the first one I checked there was doing essentially `urlparse(str | None)`. And indeed that works:

```
>>> urllib.parse.urlparse(None)
ParseResultBytes(scheme=b'', netloc=b'', path=b'', params=b'', query=b'', fragment=b'')
```

Maybe we need an overload where the first arg is specifically None?